ha/core/controller/element_controller.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
```

Comment:
Rename ha/core/controller to ha/core/controllers

Reply:
Done